### PR TITLE
Add support some run options

### PR DESCRIPTION
**Pull request by jimmyxian**
Depend: https://github.com/samalba/dockerclient/pull/228

Support following options:
+ --oom-score-adj
+ --tmpfs
+ --device-read-iops
+ --device-write-iops
+ --device-read-bps
+ --device-write-bps
+ --blkio-weight-device

Signed-off-by: Xian Chaobo <xianchaobo@huawei.com>

### CODE REVIEW

**Comment by abronan** [comment]
Sweet! Thanks @jimmyxian, seems like the test (`docker run with resources`) fails in the CI though. We should disable the tests for docker `1.9` as some of those flags are only supported since docker `1.10.x`.

It also fails for `1.10` though so I guess one of the parameter has a wrong argument.

**Comment by jimmyxian** [comment]
@abronan Yeah, you are right. I will check that. :)

**Comment by vieux** [comment]
@jimmyxian samalba/dockerclient#228 was merged :)

**Comment by jimmyxian** [comment]
@vieux Thanks. :)
I will modify the integration test.

**Comment by jimmyxian** [comment]
ping @abronan @vieux 
Done! 
BTW: There are something wrong with `--blkio-weight-device` integration test. I have removed it. :)

File: test/integration/api/run.bats [hunk 1]
```diff
@@ -99,6 +99,35 @@ function teardown() {
 	[[ "${output}" == *"someDnsOption"* ]]
 	# stop-signal
 	[[ "${output}" == *"\"StopSignal\": \"SIGKILL\""* ]]
+
+	# following options are introduced in docker 1.10, skip older version
+	run docker --version
+	if [[ "${output}" != "Docker version 1.1"* ]]; then
```
Comment (dongluochen) [inline test/integration/api/run.bats]: This skips future versions like '1.20' or '2.x'. Maybe you can skip docker 1.9 which is the only version it doesn't support.

**Comment by abronan** [comment]
Thanks @jimmyxian! LGTM when comment above is addressed.

File: test/integration/api/run.bats (hunk 1 above)
Comment (jimmyxian) [inline test/integration/api/run.bats]: OK, Thanks for your advice. Dong! :)

**Comment by jimmyxian** [comment]
@abronan Aha, will do it now.

**Comment by dongluochen** [comment]
LGTM. Thanks @jimmyxian!

**Comment by abronan** [comment]
:tada:

**Pull request merged**